examples/running-tests-using-travis-and-saucelabs/README.md

```diff
@@ -0,0 +1,9 @@
+# Example: Running Tests Using Travis CI and Sauce Labs
+
+*A set of sample files that help you learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs.*
+
+## Running the example
+
+Create a GitHub repository and copy the sample files to it.
```

Comment:
How I can copy files? You need to checkout files first and install dependencies, see https://github.com/DevExpress/testcafe/blob/master/examples/basic/README.md